app/src/main/res/layout/item_book_list.xml

```diff
@@ -23,7 +23,7 @@
             <ImageView
                 android:id="@+id/booklist_image_view"
                 android:layout_width="80dp"
-                android:layout_height="match_parent"
+                android:layout_height="100dp"
```

Comment:
restrict image height or it's gonna grow. And it seems that glide is using proportional image, i.e. conform to minimum weight/height and keep w/h ratio, so we are good here.